react/week2/todoer/src/App.js

```diff
@@ -0,0 +1,99 @@
+import "./App.css";
+import { useState, useEffect } from "react";
+
+function TodoItem({ id, description, onDelete }) {
+  const [isChecked, setIsChecked] = useState(false);
+
+  const check = () => {
+    setIsChecked(!isChecked);
+  };
+
+  return (
+    <li className={isChecked ? "checked" : "not-checked"}>
+      #{id}: {description}
+      <input type="checkbox" onChange={check}></input>
+      <button onClick={() => onDelete(id)}>Delete</button>
+    </li>
+  );
+}
+
+function TodoList({ todos }) {
+  const [stateTodos, setStateTodos] = useState(todos);
+
+  const addTodo = () => {
+      const newTodo = {
+        id: stateTodos.length > 0 ? stateTodos[stateTodos.length - 1].id + 1 : 1,
+        description: "random text",
+      };
+      const todosWithNew = stateTodos.concat(newTodo);
```

Comment:
Different way of writing this code :
 const todosWithNew =[...stateTodos,newTodo];